packages/client/src/components/Post.js

```diff
@@ -33,30 +40,40 @@ const Post = ({ title, body, isDraft, isAdmin, id, user }) => {
   const createChangeObject = (e) => {
     const { name, value } = e.target;
     setChangeObj({ ...changeObj, [name]: value });
-    updatePost({ ...changeObj, [name]: value });
+    updatePost({ ...post, [name]: value });
```

Comment:
For both of these it should probably pass the previous value to ensure that the objects stay in sync:
```suggestion
    setChangeObj(prevChangeObj => ({ ...prevChangeObj, [name]: value }));
    updatePost(prevPost => ({ ...prevPost, [name]: value }));
 ```

Reply:
gotcha!